spec/controllers/pageflow/storylines_controller_spec.rb

```diff
@@ -0,0 +1,155 @@
+require 'spec_helper'
+
+module Pageflow
+  describe StorylinesController do
+    routes { Pageflow::Engine.routes }
+
+    describe '#create' do
+      it 'creates storyline in draft' do
+        user = create(:user)
+        entry = create(:entry)
+        create(:membership, entry: entry, user: user)
+
+        expect {
+          sign_in(user)
+          aquire_edit_lock(user, entry)
+          post(:create,
+               entry_id: entry, storyline: attributes_for(:valid_storyline), format: 'json')
+        }.to change { entry.draft.storylines.count }.by(1)
+      end
+
+      it 'requires the signed in user to be member of the parent entry' do
+        user = create(:user)
+        entry = create(:entry)
+
+        sign_in user
+        post(:create, entry_id: entry, storyline: attributes_for(:valid_storyline), format: 'json')
+
+        expect(response.status).to eq(403)
+      end
+
+      it 'requires authentication' do
+        entry = create(:entry)
+
+        post(:create, entry_id: entry, storyline: attributes_for(:valid_storyline), format: 'json')
+
+        expect(response.status).to eq(401)
+      end
+    end
+
+    describe '#update' do
+      it 'updates storyline' do
+        user = create(:user)
+        entry = create(:entry)
+        storyline = create(:storyline, revision: entry.draft)
+        create(:membership, entry: entry, user: user)
+
+        sign_in user
+        aquire_edit_lock(user, entry)
+        patch(:update, id: storyline, storyline: attributes_for(:valid_storyline, configuration: {some: 'updated value'}), format: 'json')
```

Comment:
Line is too long. [138/100]<br>Space inside { missing.<br>Space inside } missing.